FrontEnd/src/tests/CompanyCard.test.js

```diff
@@ -0,0 +1,131 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import CompanyCard from '../components/SearchPage/search_field/companies/CompanyCard';
+
+afterEach(() => {
+  jest.resetAllMocks();
+});
+
+const company = {
+  id: 1,
+  name: 'Testname',
+  categories: [1, 2],
+  region: 'Testregion',
+  founded: 2005,
+  service_info: 'Testinfo',
+  address: 'Testadress',
+  banner_image: '',
+};
+
+describe('CompanyCard component unit tests', () => {
+  test('renders years of experiense', () => {
+    render(
+      <MemoryRouter>
+        <CompanyCard isAuthorized={{ isAuth: true }} companyData={company} />
+      </MemoryRouter>
+    );
+    const divElement = screen.getByText(/18 років досвіду/i, { exact: false });
+    expect(divElement).toBeInTheDocument();
+  });
+
+  test('testing stars', () => {
+    jest.mock('axios');
+    const axios = require('axios');
+
+    () => {
```

Comment:
Do we need this wrapper?

Reply:
I tried 'axios.get = jest.fn().mockResolvedValue({ ...' but it doesn't work(( The only way I've found was this wrapper